src/Education/IEducation.cs

```diff
@@ -64,11 +64,11 @@ public Education(IToolWindowService toolWindowService, IRuleMetadataProvider rul
 
         public void ShowRuleDescription(Language language, string ruleKey)
         {
-            var ruleHelp = ruleMetadataProvider.GetRuleHelp(language, ruleKey);
+            var ruleInfo = ruleMetadataProvider.GetRuleInfo(language, ruleKey);
```

Comment:
In this PR I changed the metadata provider to return `IRuleInfo`, and in the next I changed it to accept `CompositeSonarRuleId` instead of `(language, ruleKey)`.

I split the work to keep the PRs smaller.